### DISCUSSION TITLE
Setting up KERMIT for telnet over the wifi modem

### DISCUSSION THREAD

**Discussion by laffineuse** [thehighnibble/thehighnibble.github.io#117] [IMSAI8080esp replica]
Hi Group,
I am struggling with the Kermit setup to be able to telnet from the IMSAI.
When I do:
 - set port uc1
 - connect
 - ats15=1
I do get a response of "OK" for AT commands.
But when I try to dial as in ATDT a80sappleiisbbs.ddns.net:6502 (or any other telnet sites), I immediately get a "NO ANSWER".
What am I doing wrong?

**Comment by laffineuse** [comment]
Never mind! I found the solution, I had to run an STA configuration mode first, I was still in AP and therefore unable to access the internet.
Now with the correct configuration mode and the SSDI and password set, Kermit works like a charm.

**Reply by jdourish** [reply]
I am having the same problem. How did you "run an STA conf mode first"??

**Comment by udo-munk** [comment]
Is explained in detail in the documentation:
https://thehighnibble.com/imsai8080/configure/#configuring-physical-uart-parameters-speed-data-stop-bits
Section "WiFi Communications"

**Reply by jdourish** [reply]
Thanks, my IMSAI was already set to STA but I am still getting a no answer message when trying to connect to an external site using either the domain name or ip?

**Comment by udo-munk** [comment]
The site you try to reach might be down, I always test that with some PC and telnet before, if a site can be reached. Also many sites won't accept plain telnet connections anymore, only ssh encrypted sessions, and CP/M kermit can't do that, it will reject all encryption requests.

**Reply by jdourish** [reply]
Thanks, I can connect to a number of sites ok using Putty. Will try a few more and see what happens.

**Reply by BraerFox** [reply]
I know this was a while back, but Dave over at the High Nibble recommended Tera Term 5 which is what I use now for the same PuTTY reasons. Have no issues with it -and it's free. ;)

**Comment by udo-munk** [comment]
If putty gets asked about encryption by a remote host, it will answer I can do that. CP/M kermit cannot and will reject the request. This is why I use a plain telnet command line tool for such tests, because that cannot do any encryption offered, and if the host rejects that, it will fail.

**Comment by BraerFox** [comment]
been interested in telneting into some of these BBSs that still seem to be around. Just downloaded a directory, so they're still out there for the hobbyists/enthusiasts. I even kept my landline after getting a new alarm system.

I have an old Atari 800 with an 810 modem, as well as a working landline. 

But would love to incorporate my IMSAI 8080 if possible
Anyone out here mess with this?

I'm guessing I should probably emulate via the wifi modem first, before I move on to old hardware -if it's even possible. Just thought i might be kind of fun.

Thanks!
